src/rules/functionality.ts

```diff
@@ -93,6 +101,8 @@ export default {
   'triple-equals': [true, 'allow-null-check'],
   // Makes sure result of `typeof` is compared to correct string values
   'typeof-compare': true,
+  // Warns if an explicitly specified type argument is the default for that type parameter. <FEEDBACK>
+  'use-default-type-parameter': false,
```

Comment:
I'd leave it off, I don't see why specifying it as the default should really matter (and will likely make it more clear at the call site)